tests/vmi_lifecycle_test.go

```diff
@@ -223,7 +223,8 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			var err error
 			vmi := tests.NewRandomVMI()
 			vmi.Labels = map[string]string{
-				"debugLogs": "true",
+				"debugLogs":    "true",
+				"logVerbosity": "8",
```

Comment:
Since raising the log verbosity that much will automatically enable libvirt logs, I think you can replace the existing annotation instead of adding to it.

Reply:
Done, thanks!

BTW - I needed to fix a tiny bug in `renderLaunchManifest` for that to work. Turns out that the verbosity haven't been updated properly when it was set though labels.